martian-lab/tests/error_test/expected/SUM_SQUARES/SUM_SQUARES/fork0/chnk3/_errors

```diff
@@ -0,0 +1 @@
+hit special failure value
```

Comment:
It would be better if the martian adapter can append a line saying `The traceback is written out in _stackvars`

Reply:
Hmm - I'm not sure if we want that in every customer facing output string. So based on Adam's theory that customers will not be able to do anything with a stack trace I'm inclined to leave that out. (we'll have to train QA/SFO/CB to know where to look)